kart/exceptions.py

```diff
@@ -21,6 +21,8 @@
 SPATIAL_FILTER_PK_CONFLICT = 27
 INVALID_FILE_FORMAT = 28
 UNCOMMITTED_CHANGES = 29
+# Ran out of 2x numbers. Oh well.
+CONFLICTING_POSSIBILITIES = 31
```

Comment:
I'm not really fond of this name, doesn't seem very descriptive of what it actually is. It's a conflict at commit time, maybe it shoudl be `AMBIGUOUS_COMMIT` or `WORKING_COPY_CONFLICT` or something

Reply:
Landed on WORKING_COPY_OR_IMPORT_CONFLICT. It's long but it works